.github/workflows/beam_PostCommit_Java_Examples_Dataflow_ARM.yml

```diff
@@ -111,7 +111,7 @@ jobs:
             gradle-command: :runners:google-cloud-dataflow-java:arm:examplesJavaRunnerV2IntegrationTestARM
             max-workers: 12
             arguments: |
-                -PtestJavaVersion${{ matrix.java_version }} \
+                -PtestJavaVersion=${{ matrix.java_version }} \
```

Comment:
fixed a few typo introduced in #28833 this caused the test actually run on default java version